api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirConceptSourceServiceImpl.java

```diff
@@ -24,17 +24,17 @@
 import org.openmrs.module.fhir2.api.FhirConceptSourceService;
 import org.openmrs.module.fhir2.api.dao.FhirConceptSourceDao;
 import org.openmrs.module.fhir2.model.FhirConceptSource;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.Cacheable;
 
-@Component
 public class FhirConceptSourceServiceImpl implements FhirConceptSourceService {
```

Comment:
Could you please add implements OpenmrsService to FhirConceptSourceService for consistency?